### PR TITLE
ENDOC-119 Add protected assets notes and cleanup jacms tags

### PR DESCRIPTION
**Pull request by joewhite101**
This is a starting point for creating protected assets. Probably needs another pass with some depth on rendering those assets into lists but involves some JSP tag development documentation that is out of scope for this story (i think).

### CODE REVIEW

**Comment by joewhite101** [comment]
Published to stage:
https://entando.github.io/entando-docs/next/tutorials/cms/creating-protected-resources.html

**Review by nshaw — CHANGES_REQUESTED** [review]
See https://github.com/entando/entando-docs/pull/220 for my requested changes.

File: vuepress/docs/next/tutorials/cms/creating-protected-resources.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Select `Users -> Groups` from the left navigation
+2. Select `Add`
+3. Enter a `Name` and a `Code` for your group
+
+The groups you create here can be utilized to protect pages, content, images, and attachments in the CMS. Groups can be assigned to individual users in the `Users` section of the app builder or as part of a customization of your entando-core-app using APIs or custom code.
+
+## Protected pages
+
+1. Create a new page. Assign values as you see fit. (see here if you need steps to create a page TODO).
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/next/tutorials/cms/creating-protected-resources.md:19]: I went ahead and dropped this TODO for now.

[the remark this answers is not in the document]
Comment (joewhite101) [reply vuepress/docs/next/tutorials/cms/creating-protected-resources.md:19]: Oops. Meant to remove that. Thanks

**Review by nshaw — APPROVED** [review]

**Pull request merged**